src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -432,7 +432,7 @@ def update_ag_backend_http_settings_collection(instance, parent, item_name, port
         instance.request_timeout = timeout
     if connection_draining_timeout is not None:
         instance.connection_draining.enabled = bool(connection_draining_timeout)
-        instance.connection_draining.drain_timeout_in_sec = connection_draining_timeout
+        instance.connection_draining.drain_timeout_in_sec = connection_draining_timeout or 1
```

Comment:
Yes. The service will say that it has to be greater than 1 (though ironically it only has to be equal to 1).